installer/templates/phx_single/config/dev.exs

```diff
@@ -48,8 +48,7 @@ config :<%= @app_name %>, <%= @endpoint_module %>,
     patterns: [
       ~r"priv/static/.*(js|css|png|jpeg|jpg|gif|svg)$",<%= if @gettext do %>
       ~r"priv/gettext/.*(po)$",<% end %>
-      ~r"lib/<%= @lib_web_name %>/(live|views)/.*(ex)$",
-      ~r"lib/<%= @lib_web_name %>/templates/.*(eex)$"
+      ~r"lib/<%= @lib_web_name %>/(controllers|live|components)/.*(ex|heex)$"
```

Comment:
```suggestion
      ~r"lib/<%= @lib_web_name %>/(controllers|live|components)/.*(ex)$"
```

Reply:
I got your point - `.*(ex)` will match `.ex` file and `.heex` file. 

But I prefer to use `.*(ex|heex)`, which **tells the user explicitly that we are matching `.ex` file and `.heex` file.**

If you think `.*(ex|heex)` is making the regexp redundant. Maybe we can use a explicit one, like `.*\.(ex|heex)`. Then the final suggestion is:
```elixir
~r"lib/<%= @lib_web_name %>/(controllers|live|components)/.*\.(ex|heex)$"
```
What do you think?

Reply:
For code consistency, it is also possible to adjust other two lines:
```elixir
~r"priv/static/.*\.(js|css|png|jpeg|jpg|gif|svg)$",<%= if @gettext do %>
~r"priv/gettext/.*\.(po)$",<% end %>
```

Reply:
I prefer the simpler regex, and if defining both `ex` and `heex` seems better for clarity then I would vote to leave it as-is, meaning ignore my suggestions :)